tensorflow_cloud/containerize.py

```diff
@@ -40,6 +40,7 @@
     VERSION = "latest"
 
 
```

Comment:
Can we add a comment explaining why we need this ?

Reply:
This what?  Line 40 or Line 43?

Reply:
Line 43

Reply:
On more thought, this should be moved before the usage in line 137. Comment can be "Updating the name for RC's to match with the TF generated RC docker image names."

Reply:
Done but I don't know where this incompatibility was generated. I cannot see who set `_TAG_PREFIX` in the google infra for  https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/dockerfiles/assembler.py

Reply:
Yeah, let me try to find it.

Reply:
it matches the version name for the Python releases (Since Python package versions can't contain hyphens.). Images are generated by this: google3/learning/brain/testing/kokoro/docker/nightly.sh